test/test_excepthook.py

```diff
@@ -1,12 +0,0 @@
-import logging
```

Comment:
Why did you remove this test?

Reply:
We need to verify that 'exchook' is installed, but not that it is actually called (because it is responsibility of Python itself). So there is no need to spawn external process etc, I've simplified that test.

Reply:
Ok, 👍